phpBB/phpbb/install/helper/iohandler/cli_iohandler.php

```diff
@@ -0,0 +1,265 @@
+<?php
```

Comment:
For the record: revamp the progress bar to do like in the reparsers (always use a progress bar, just change the format)